src/app/zap-templates/zcl/data-model/chip/drlc-cluster.xml

```diff
@@ -139,70 +121,39 @@ limitations under the License.
     <item fieldId="6" name="HeatingSourceControl" type="HeatingSourceControlStruct" optional="true"/>
   </struct>
 
-  <!-- This does not quite match the spec, which has LoadControlEventStruct as a
-       fabric-scoped struct with some fabric-sensitive fields, but that doesn't
-       really make sense in all the places where the struct is used, and the spec
-       does not really define other things in a way that would work with that at
-       the moment. -->
```

Comment:
Why is this comment being removed?  Certainly the XML here is still not matching the spec (which has the fabric-scoped and fabric-sensitive bits going on).